src/ansys/templates/python/solution/{{cookiecutter.__project_name_slug}}/pyproject.toml

```diff
@@ -43,10 +43,10 @@ priority = "supplemental"
 [tool.poetry.dependencies]
 python = ">=3.8, <3.11"
 ansys-saf-portal = {version = "1.0rc3", source = "solutions-private-pypi"}
```

Comment:
I have pointed this out a few times - I'm not sure what the portal dev cycle is, but there hasn't been a release in months and theoretically it should also be a stable 1.0.0 release to match GLOW.

Reply:
I think Portal and Glow-Engine are not necessarily following the release cycle of saf-desktop. So I am not expecting to have a saf-portal 1.0 together with saf-desktop 1.0. I asked the Portal Team and the saf-portal 1.0.0 will be released in about three weeks. So I suggest we upgrade saf-portal version in the solution template in a future PR.

Reply:
Sounds good - as a follow-up, can you please confirm that the packages you mentioned will all provide a list of supported versions?

For example, in a saf-portal release, I would expect an easy place I can go see that `this release of saf-portal is compatible with saf-desktop versions XXX, YYY, and ZZZ` or similar

I just want to make sure we are clear about the dependencies because historically GLOW has required very specific portal versions to be compatible. The same probably applies to our actions versions, which we are also discussing in a different thread.